pvlib/temperature.py

```diff
@@ -393,19 +383,21 @@ def pvsyst_cell(poa_global, temp_air, wind_speed=1.0, u_c=29.0, u_v=0.0,
     air temperature :math:`T_{a}` (C) and wind speed :math:`WS` (m/s). Model
     output is cell temperature :math:`T_{C}`. Model parameters depend both on
     the module construction and its mounting. Parameters are provided in
-    [1]_ for open (freestanding) and close (insulated) mounting configurations,
-    , and are coded for convenience in
+    [1]_ for open (freestanding), close (insulated), and intermediate
+    (semi_integrate) mounting configurations, and are coded for convenience in
```

Comment:
```suggestion
    (semi_integrated) mounting configurations, and are coded for convenience in
```

Reply:
This typo is already fixed in the latest commits